dali/python/nvidia/dali/plugin/numba/experimental.py

```diff
@@ -311,6 +316,11 @@ def __call__(self, *inputs, **kwargs):
         return outputs[0] if len(outputs) == 1 else outputs
 
     def __init__(self, run_fn, out_types, in_types, outs_ndim, ins_ndim, setup_fn=None, device='cpu', batch_processing=False, blocks=None, threads_per_block=None, **kwargs):
+        toolkit_version = cuda.runtime.get_version()
+        driver_version = cuda.driver.driver.get_version()
+
+        assert toolkit_version <= driver_version, f"Environment is not compatible with Numba GPU operator. Driver version is {driver_version} and CUDA Toolkit version is {toolkit_version}. Driver can not be older than the CUDA Toolkit"
```

Comment:
As always, one can disable asserts, it should be a Runtime Error raised here.

Reply:
Done